fdbsyncd/fdbsync.cpp

```diff
@@ -43,6 +43,62 @@ FdbSync::~FdbSync()
     }
 }
 
+
+// Check if interface entries are restored in kernel
+bool FdbSync::isIntfRestoreDone()

```

Comment:
FdbSync::isIntfRestoreDone() ,  FdbSync::isReadyToReconcile() and RouteSync::isReadyToReconcile() are doing similar tasks, it seems we could make them into library calls with different input parameters.  Later PR is probably fine.

Reply:
sure .. will consider in future updates to the code